wordpress-mobile/WordPress-iOS
# Quick Start v2: VoiceOver support missing on Next Steps

Fixes #12964 

This PR updates the Next Steps section so that the three dots button (more menu) and the two cells (Customize Your Site and Grow Your Audience) have VoiceOver support. It also adds accessibility hints as an optional property for BlogDetailRow, and if it exists, is read for BlogDetailsViewController's tableview rows.

<a href="https://user-images.githubusercontent.com/1062444/84310145-cc205780-ab26-11ea-8996-ced2e473c467.PNG"><img src="https://user-images.githubusercontent.com/1062444/84310145-cc205780-ab26-11ea-8996-ced2e473c467.PNG" width="350" /></a>

### To test:
Pre-requisite: build this branch on your test device.

1. Create a new site. Switch site > + button > Create WordPress.com site (follow the prompts).
2. When the fancy alert asks, "Want a little help getting started?" select "Yes, Help Me" to activate the Quick Start guide.
3. Close the "Customize Your Site" modal view (X button in the top left).
4. Turn on VoiceOver ("hey Siri turn on VoiceOver")
5. Tap the Media button then two-finger swipe down to get VoiceOver to read the section header, more menu button, and two cells in the Next Steps section. It should read: `Next Steps. More menu (button). Customize your site. N of N completed. Button. (pause). A series of steps showing you how to add a theme, site icon and more.` and `Grow your audience. N of N completed. Button. (pause).  A series of steps to assist with growing your site's audience.`

PR submission checklist:
